tests/appsec/iast/source/test_multipart.py

```diff
@@ -13,5 +13,5 @@ class TestMultipart(BaseSourceTest):
     endpoint = "/iast/source/multipart/test"
     requests_kwargs = [{"method": "POST", "files": {"file1": ("file1", "bsldhkuqwgervf")}}]
     source_type = "http.request.multipart.parameter"
-    source_name = "Content-Disposition"
+    source_name = {"name", "Content-Disposition"}
```

Comment:
```suggestion
    source_names = {"name", "Content-Disposition"}
```

replace any occurence of `source_name` to `sources_names`, and change the value to a list

Reply:
applied